wrapper/src/main/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPlugin.java

```diff
@@ -175,7 +176,12 @@ public AwsSecretsManagerConnectionPlugin(final PluginService pluginService, fina
 
     this.secretKey = Pair.of(secretId, region);
 
+    ConfigurationProfile profile = this.pluginService.getConfigurationProfile();
```

Comment:
Does this block need to be added to the OktaAuth or FederatedAuth plugins? I noticed they use credentials providers, but they do not use AwsCredentialsManager, instead they use a CredentialsProviderFactory. The only implementation o f CredentialsProviderFactory always uses AnonymousCredentialsProvider. Should these plugins be using the credentials provider defined by either the ConfigurationProfile or the AwsCredentialsManager?

Reply:
talked with Karen and Aaron and they said Okta and Federated Auth should not be using the custom handler